### DISCUSSION TITLE
Add official Discord channel to the website

### DISCUSSION THREAD

**Discussion by dylanschultzie** [oasisprotocol/community-forum#28] [General]
There isn't currently a unified location for the community to gather. 

There exists:
1. (official) Oasis Telegram: https://t.me/oasisprotocolcommunity
2. (official) Oasis Discord: https://discord.gg/npnyfGpU7W
3. (official) Slack channel: https://join.slack.com/t/oasiscommunity/shared_invite/zt-rrfh9vlu-Sn_IWDI8oKw1SABI5caKMw
4. (unofficial) Discord: https://discord.gg/4Wj7YpfvD3

However, neither Discord channel is listed on the website:

Bottom banner: 
![image](https://user-images.githubusercontent.com/9121234/131557538-9898dfab-74c3-454c-b855-ced2610ef01d.png)

Community page:
![image](https://user-images.githubusercontent.com/9121234/131557584-7122833d-1793-426d-995d-7e6cfad61594.png)

The official Discord *is* listed within the documentation, but you have to dig to find it: https://docs.oasis.dev/general/oasis-network/connect-with-us

**Comment by Afauvre** [comment]
@dylanschultzie do you think Discord is a more effective convening site than Slack?

**Comment by dylanschultzie** [comment]
@Afauvre yes, without a doubt. Discord, in my experience, is the standard for crypto communities. There are upwards of a dozen new members joining per day, but there isn't any support/help provided. 

![image](https://user-images.githubusercontent.com/9121234/131720996-7e0173d9-124d-4d9e-975b-ecaad798b5d2.png)

In my experience, the best thing you could do to support the Oasis community's growth is:
1. **put it on the website**
2. Have active moderators/personnel in the channels to answer questions and provide support. 
3. Add roles to the server to help encourage activity (who runs a validator? who's an ambassador? who's senior?)

**Comment by dylanschultzie** [comment]
In addition, the Slack channel also doesn't appear to be particularly active. I posted yesterday and received no response:  

![image](https://user-images.githubusercontent.com/9121234/131721949-92221d00-7ab5-49db-becd-951ecbcfa84a.png)

**Comment by Afauvre** [comment]
Thanks this is helpful! Yes definitely seems like a good idea to add Discord to the website! And make it a more official channel in that case

**Reply by dylanschultzie** [reply]
To be clear, I don't mean to fault anyone - just expressing what feels like the reality to me.
